tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -1163,6 +1179,38 @@ bool IsMatchedMatMulBiasAddAndGeluExact(
   using utils::MatchingDirection;
   using utils::NodeStatus;
   // clang-format off
+  static utils::OpTypePattern* gelu_exact_pattern2 = new utils::OpTypePattern
+    {"Mul", "output", NodeStatus::kReplace,
+      {
+        {"Add|AddV2", "erf_plus_one", NodeStatus::kRemove,
+          {
+            {"Erf", "erf", NodeStatus::kRemove,
+              {
+                {"Mul", "bias_add_times_square_root_one_half", NodeStatus::kRemove,
+                  {
+                    {"BiasAdd", "bias_add", NodeStatus::kRemove},
+                    {"Cast|Const", "square_root_one_half", NodeStatus::kRemain}
+                  }
+                }
+              }
+            },
+            {"Cast|Const", "one", NodeStatus::kRemain}
+          }
+        },
+        {"Mul", "erf_plus_one_times_one_half", NodeStatus::kRemove,
+          {
+            {"BiasAdd", "bias_add", NodeStatus::kRemove,
+              {
+                {"MatMul", "matmul", NodeStatus::kRemove},
+                {"*", "bias", NodeStatus::kRemain}
+              }
+            },
+            {"Cast|Const", "one_half", NodeStatus::kRemain}
+          }
+        }
+      }
+    };
+
   static utils::OpTypePattern* gelu_exact_pattern = new utils::OpTypePattern
```

Comment:
Please add a comment drawing the subgraph, i.e.,
```c++
  // Pattern 1:
  //    Const: 1/sqrt(2)        Const: 1    Const: 1/2
  //                  \               \         \
  //  * --> BiasAdd --> Mul --> Erf --> AddV2 --> Mul --> Mul
  //      /         \___________________________________/
  // MatMul
```

Reply:
Thanks! this is very useful.  Added in the file